client/src/state/Run/RunAPI.ts

```diff
@@ -3,16 +3,18 @@ import superagent from 'superagent';
 import { IRunState } from './RunTypes';
 
 export async function getRuns(): Promise<IRunState> {
-    return (await superagent.get(`/api/v1/run`)).body;
+    const body = (await superagent.get(`/api/run/api/v1/run`)).body;
```

Comment:
The URL looks a bit redundant

Reply:
this was due to the proxy.. but the proxy is removed now.